openarc/openarc.c

```diff
@@ -3375,6 +3379,7 @@ mlfi_eom(SMFICTX *ctx)
 	Header hdr;
 	struct authres ar;
 	unsigned char header[ARC_MAXHEADER + 1];
+	u_char arcchainbuf[ARC_MAXHEADER + 1];
```

Comment:
We should use "unsigned char" on both rather than mixing, but I'm not going to send this back again.